includes/Modules/Idea_Hub.php

```diff
@@ -312,9 +312,14 @@ private function admin_notice_idea_hub_ideas( $notices ) {
 			array(
 				'content'         => function() use ( $escape_and_wrap_notice_content ) {
 					$message = sprintf(
-						/* translators: %s: URL to saved ideas */
-						__( 'Site Kit by Google: Want some inspiration for a new post? <a href="%s">Revisit your saved ideas</a> in Site Kit.', 'google-site-kit' ),
-						esc_url( $this->context->admin_url( 'dashboard', array( 'idea-hub-tab' => 'saved-ideas' ) ) )
+						/* translators: %1$s: Plugin name. %2$s: Notice content. */
+						__( '%1$s: %2$s', 'google-site-kit' ),
+						get_plugin_data( GOOGLESITEKIT_PLUGIN_MAIN_FILE )['Name'],
+						sprintf(
```

Comment:
Nested `sprintf` calls here to keep the original translated notice text as it was. Feedback is very welcome if this should be addressed with a different approach.

Reply:
This is the correct end result but makes it a bit harder to follow, particularly due to all the positional arguments and long lines.

How about we move the nested `sprintf` above as `$message_body`? I'm open to other ideas for what to call it 🙂

Reply:
Sorted.